### PR TITLE
Mark 1.0 as unsupported and 1.1 as soon-unsupported in docs

### LINKED ISSUE #4430: Mark 1.0 as unsupported in docs
@bdarnell says we support 2 version prior to stable. So currently that means 2.1, 2.0, and 1.1. 
For all earlier versions, we should put a disclaimer/warning all all pages.

### PR DESCRIPTION
**Pull request by jseldess**
Fixes #4430.

screen shot 2019-02-25 at 10 49 13 pm

### CODE REVIEW

**Comment by jseldess** [comment]
@bdarnell, is this warning sufficient, or do we need to be more explicit about our rule of supporting 2 version prior to stable?

File: _includes/v1.0/misc/unsupported-version.md [hunk 1]
```diff
@@ -0,0 +1,3 @@
+{{site.data.alerts.callout_danger}}
+This version of CockroachDB is no longer supported. Please use a more recent version.
```
Comment (sploiselle) [inline _includes/v1.0/misc/unsupported-version.md]: I think that including a link here to the install/download page for `/stable` would feel like we were trying to help users who're trying to use these docs.

**Comment by bdarnell** [comment]
> is this warning sufficient, or do we need to be more explicit about our rule of supporting 2 version prior to stable?

It would be nice to document the support policy somewhere, but that can be a separate change. (I'm not sure where we'd put it currently)

Maybe we should add a warning to 1.1 too. Not quite as scary (yellow instead of red), but say that 1.1 will no longer be supported after the spring 2019 release.

File: _includes/v1.0/misc/unsupported-version.md (hunk 1 above)
Comment (jseldess) [reply _includes/v1.0/misc/unsupported-version.md]: Done.

**Comment by jseldess** [comment]
@bdarnell, updated the unsupported callout and added a soon-unsupported callout to 1.1 docs. PTAL.

**Pull request merged**